plugins/events-backend/src/service/DefaultEventBroker.ts

```diff
@@ -56,12 +62,17 @@ export class DefaultEventBroker implements EventBroker {
         }
       }),
     );
+    // This allows to listen to events also in the frontend
+    // TODO(drodil): The actual events frontend to subscribe to the topics
+    await this.signalsService?.publish(params.eventPayload, {
```

Comment:
I think this should still be discussed; should all the backend-events also be distributed to the frontend? Or is this in the plugin's own logic to publish first to the backend and then to the frontend? Opinions @Rugvip and other maintainers? My gut feeling is that this line should not be here as some of the backend events contain information that should be visible only to certain users.